packages/web-app-user-management/src/components/Users/DeleteUserModal.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
```

Comment:
In theory this wouldn't need a wrapping div since it has only one root node anyways?